service/index.js

```diff
@@ -53,17 +53,22 @@ async function init() {
 }
 
 function importFromLucidchart(fileName, callback) {
-  const importFile = `./data/${fileName}.csv`;
-  console.log(`Importing file: ${importFile}`);
-  preParser.newCollection();
-  fs.createReadStream(importFile).pipe(csv()).on('data', (rowData) => {
-    preParser.have(rowData);
-  }).on('end', async () => { // We are done pulling in data
-    if (await preParser.prepare(knex)) {
-      await dbBuilder.make(preParser.lucidCollectionPreped, knex);
-    }
-    callback();
-  });
+  if(isViolateRule(fileName)) {
```

Comment:
It's not the filename we want to apply restrictions on, it's the journey name itself.
If you open up any Lucidchart with an Envoy journey and click on the yellow swim-lane, then look at the data, you can see an attribute called 'name'. It's this that needs checking, as it's this that's used as the journey name when the data is imported, and becomes a key in the JSONB object. Please feel free to drop by my desk if you like and I can show you what I mean (it might save a lot of typing!)